x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/X-FallibleArrayState.java.st

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.compute.aggregation;
+
+import org.elasticsearch.common.util.BigArrays;
+$if(boolean)$
+import org.elasticsearch.common.util.BitArray;
+$else$
+import org.elasticsearch.common.util.$Type$Array;
+$endif$
+import org.elasticsearch.compute.data.Block;
+$if(long)$
+import org.elasticsearch.compute.data.IntVector;
+$endif$
+import org.elasticsearch.compute.data.$Type$Block;
+$if(int)$
+import org.elasticsearch.compute.data.$Type$Vector;
+$endif$
+$if(boolean||double||float)$
+import org.elasticsearch.compute.data.IntVector;
+$endif$
```

Comment:
Again, I'm sorry for this silly dance we have to do for imports. I just couldn't figure out a way to turn off the import checkstyle rule for just this file.

Reply:
This is a copypaste+failed things, I didn't touch anything here :hehe:

I'll diff them in case it's worth like, adding an extra parameter to generate both (fallible and non fallible) from the same template. But I'll try to not overcomplicate it, I don't want 2 levels of templating here 😆